Dockerfile.dapper

```diff
@@ -16,6 +16,7 @@ ENV DAPPER_SOURCE /go/src/github.com/longhorn/longhorn-engine
 WORKDIR ${DAPPER_SOURCE}
 
 RUN zypper -n addrepo --refresh https://download.opensuse.org/repositories/system:/snappy/SLE_15/system:snappy.repo && \
+    zypper -n addrepo --refresh https://download.opensuse.org/repositories/devel:tools:scm/15.5/devel:tools:scm.repo && \
```

Comment:
Do you know why we need this repo? is any package unable to install w/o this repo?

Reply:
This was part of migration process for 15.3 to 15.4. It's building with/without these. Let me remove this and update PR.